mkosi.presets/00-base/mkosi.build

```diff
@@ -193,3 +193,14 @@ if [ "$WITH_TESTS" = 1 ]; then
 fi
 
 ( set -x; meson install -C "$BUILDDIR" --quiet --no-rebuild --only-changed )
+
+# Ensure that side-loaded PE addons are loaded if signed, and ignored if not
+if [ -d "${DESTDIR}/boot/loader" ]; then
+    mkdir -p "${DESTDIR}/boot/loader/addons"
+    ukify --secureboot-private-key mkosi.secure-boot.key --secureboot-certificate mkosi.secure-boot.crt --cmdline this_should_be_here -o "${DESTDIR}/boot/loader/addons/good.addon.efi"
+    ukify --cmdline this_should_not_be_here -o "${DESTDIR}/boot/loader/addons/bad.addon.efi"
+elif [ -d "${DESTDIR}/efi/loader" ]; then
+    mkdir -p "${DESTDIR}/efi/loader/addons"
+    ukify --secureboot-private-key mkosi.secure-boot.key --secureboot-certificate mkosi.secure-boot.crt --cmdline this_should_be_here -o "${DESTDIR}/efi/loader/addons/good.addon.efi"
+    ukify --cmdline this_should_not_be_here -o "${DESTDIR}/efi/loader/addons/bad.addon.efi"
+fi
```

Comment:
Why not use a var for `/boot` vs `/efi` here? Then the branches can be unified, no?

Reply:
Why have the logic for /boot at all? We can just unconditionally use /efi no?

Reply:
someone might override the local config, doesn't cost much to support both